### PR TITLE
Notes for chapter 1

### PR DESCRIPTION
**Pull request by mjg123**
these can be polished more and more to a shiny chapter of the book -
  and also used as a basis for lecture 1

### CODE REVIEW

File: content/book/chapter_1/Chapter_1.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+### Not everyone is using your computer
+
+(so we can't use localhost forever)
+
+### Accessing from the internet
+
+Your computer is probably not directly connected to the internet, but connects through your router or similar at home:
+
+![home server architecture](./webapp3.png)
```
Comment (mjg123) [inline content/book/chapter_1/Chapter_1.md]: This picture and the next have the browser in the wrong place. It should be connecting from the outside through the router.

File: content/book/chapter_1/Chapter_1.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+TODO: Mention DNS here
+
+To mitigate this in 2008, you would have rented some space - about the size of a pizza box - in a datacenter. Then drive there with a small computer and leave it there.:
+
+![basic DC arch](./webapp4.png)
+
+Todo: pictures and stats about DCs
+
+Problems which you still have:
```
Comment (iovana) [inline content/book/chapter_1/Chapter_1.md]: Add a bit more info here on what the issues are, but might not necessarily want to go into too much detail about this in the lecture.

File: content/book/chapter_1/Chapter_1.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  - Scaling
+  - Updating your OS
+  - Updating your application
+  - Paying for it
+  
+
+## What next?
+
+The rest of this course is about the improvements made in the last decade or so to this state of affairs. The biggest change has been the popularization of "The Cloud". But always remember:
+
```
Comment (iovana) [inline content/book/chapter_1/Chapter_1.md]: Are we going to give a quick overview of what the next week's lecture is about?

File: content/book/chapter_1/Chapter_1.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+  - the wifi connection in your house might fail
+  - your computer might get turned off by a cat
+  - your ISP might decide to issue your router with a different IP address
+  - your house might have a power cut
+  - your ISP might not not let you have a very high upload speed
+  
+etc
+
+TODO: Mention DNS here
```
Comment (mjg123) [inline content/book/chapter_1/Chapter_1.md]: Yes this is needed

File: content/book/chapter_1/Chapter_1.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+![home server architecture](./webapp3.png)
+
+Configure router to forward connections, then you can use external IP to access your application at `http://external-ip:8080/` but this isn't ideal for a number of reasons:
+
+  - the wifi connection in your house might fail
+  - your computer might get turned off by a cat
+  - your ISP might decide to issue your router with a different IP address
+  - your house might have a power cut
+  - your ISP might not not let you have a very high upload speed
```
Comment (mjg123) [inline content/book/chapter_1/Chapter_1.md]: AUTH AUTH AUTH here. we need to have auth at this point.

**Pull request merged**